app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,33 @@
+class CommentsController < ApplicationController
+  before_action :find_user
+  before_action :find_post
+
+  def new
+    @comment = @post.comments.new
+  end
+
+  def create
+    @comment = @post.comments.new(comment_params)
+    @comment.author = @user
+    if @comment.save
+      flash[:notice] = 'Comment created successfully.'
+      redirect_to user_post_path(@user, @post)
+    else
+      render 'new'
+    end
+  end
```

Comment:
- [ ] Please kindly give attention to this aspect, I was unble to create a comment; I got the error message below 👇 

<img width="673" alt="Untitled" src="https://github.com/Young-Gaius/Rails-Blog-App/assets/103386183/b5657f84-1c73-4743-9345-e4dc98c828ab">